.travis.yml

```diff
@@ -31,6 +35,7 @@ notifications:
 
 before_install:
   - pip install $CHAINER_INSTALL chainer
+  - if [[ $ONNX_VER != "" ]]; then pip install onnx==$ONNX_VER; fi
   - pip install -U -e .[travis]
```

Comment:
What will happen if we change the requirement in setup.py not to include onnx 1.4.1 (e.g., onnx>=1.5.0) but forget to change this file? I guess onnx 1.5.0 will be installed and we'll run tests with an unexpected version of ONNX? If this is the case, is there any way to make sure we are using $ONNX_VER? I guess we can swap the two `pip install`s?

Reply:
Your point is exact.

Unfortunately, when ONNX-Chainer's setup.py drops ONNX1.4.1 then install it, error log is out but pip installation will succeed. 

```bash
$ pip install -e .  # drops ONNX 1.4.1
$ pip install onnx==1.4.1  # logged error but exit 0
...
ERROR: onnx-chainer 1.4.1 has requirement onnx<1.6,>=1.5.0, but you'll have onnx 1.4.1 which is incompatible.
Installing collected packages: onnx
  Found existing installation: onnx 1.5.0
    Uninstalling onnx-1.5.0:
      Successfully uninstalled onnx-1.5.0
Successfully installed onnx-1.4.1
$
```

However, at least, test env is expected and we can catch test fails caused by ONNX version.